### PR TITLE
Fix C-API sample. Update Issue template.

### PR DESCRIPTION
**Pull request by jignparm**
C-API sample broke after function signature change.

Added Visual Studio version to GitHub issue template. In case users are on older version of VS.

### CODE REVIEW

File: csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp [hunk 1]
```diff
@@ -57,13 +57,13 @@ int main(int argc, char* argv[]) {
...
   std::vector<const char*> input_node_names(num_input_nodes);
-  std::vector<size_t> input_node_dims;  // simplify... this model has only 1 input node {1, 3, 224, 224}.
+  std::vector<int64_t> input_node_dims;  // simplify... this model has only 1 input node {1, 3, 224, 224}.
                                         // Otherwise need vector<vector<>>
 
   printf("Number of inputs = %zu\n", num_input_nodes);
 
   // iterate over all input nodes
-  for (int i = 0; i < num_input_nodes; i++) {
+  for (unsigned int i = 0; i < num_input_nodes; i++) {
```
Comment (snnn) [inline csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp]: Why it's unsigned int?

Comment (jignparm) [reply csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp]: Switched back to int. Was initially getting warnings about signed/unsigned comparison -- but can't reproduce them.

Comment (snnn) [reply csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp]: Why don't use size_t?

Comment (jignparm) [reply csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp]: that works too. updated.

**Pull request merged**